tortoise/__init__.py

```diff
@@ -163,7 +163,7 @@ def _discover_models(cls, models_path, app_label) -> List[Type[Model]]:
             connection = client_class(**db_params)
             if create_db:
                 await connection.db_create()
-            await connection.create_connection()
+            await connection.create_connection(True)
```

Comment:
I think it's better to pass it as a kwarg, because right now it's not obvious what this `True` does and it forces reader to jump into function to see why we are passing `True` here

Reply:
:+1: